root/layout/index.js

```diff
@@ -305,7 +305,9 @@ const Layout = ({
           ? <MergeHelper merger={$c.session.merger} />
           : null}
 
-        <Footer />
+        <div className="bs">
+          <Footer />
+        </div>
```

Comment:
It seems like we can add the `bs` class inside `Footer` and not have to wrap it in a div.

Reply:
Done